tests/check_consistent.py

```diff
@@ -19,15 +19,8 @@
 import tomli
 
 consistent_files = [{"stdlib/@python2/builtins.pyi", "stdlib/@python2/__builtin__.pyi"}]
-metadata_keys = {
-    "version",
-    "requires",
-    "extra_description",
-    "obsolete_since",
-    "no_longer_updated",
-    "stubtest",
-    "stubtest_apt_dependencies",
-}
+metadata_keys = {"version", "requires", "extra_description", "obsolete_since", "no_longer_updated", "tool"}
+stubtest_keys = {"skip", "apt_dependencies"}
```

Comment:
I think we should convert this to a dict of sets:

```suggestion
tool_keys = {"stubtest": {"skip", "apt_dependencies"}}
```

and then ...